js/reject-feedback.js

```diff
@@ -150,7 +150,15 @@
 					div.find( 'textarea[name="feedback_comment"]' ).val( '' );
 				}
 			}
-		);
+		).fail(
+			function( xhr, msg ) {
+				msg = 'An error has occurred';
+				if ( xhr.responseText ) {
+					msg += ': ' + xhr.responseText;
+				}
+				msg += '. Please, take a screenshot, send it to the developers, and reload the page to see if it still worked.';
+				$gp.notices.error( msg );
+			} );
```

Comment:
I think there is this little whitespace missing:
```suggestion
			}
		);
```